src/store/messages/api.ts

```diff
@@ -46,10 +46,11 @@ export async function editMessageApi(
   channelId: string,
   messageId: number,
   message: string,
-  mentionedUserIds: string[]
+  mentionedUserIds: string[],
+  data?: object
```

Comment:
what is data ? what is the structure of it that we are using in zOS ?

Reply:
I understood it's like optional `options` which it contains `hidePreview`, le's add the type please